src/ViewRendererBundle/objects/helper/TPkgViewRendererLessCompiler.php

```diff
@@ -270,4 +283,15 @@ private function getMinifyParameter()
         return \ChameleonSystem\CoreBundle\ServiceLocator::getParameter(
             'chameleon_system_core.resources.enable_external_resource_collection_minify');
     }
+
+    private function checkAndCreateDir(string $dir): bool
```

Comment:
Could we rename this method to something like `createOutputDirIfRequired`? The method's purpose isn't really to check anything but to assure that the directory exists.

Also I'd remove the return value and simply throw an exception instead of returning false. Then `getGeneratedCssForPortal` could simply call the method without having to check the result.

Reply:
ok, ok